.readthedocs.yaml

```diff
@@ -0,0 +1,18 @@
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.9"
+
+sphinx:
+  configuration: doc/conf.py
+
+formats:
+  - pdf
```

Comment:
Do we need these format?

Reply:
they were enabled in the old setup. We can disable them if not needed.

Reply:
I see. Let's keep them then.